examples-per-language/php/behavioral/Command.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+require_once __DIR__ . '/autoload.php';
```

Comment:
That is true, but as far as I'm concerned, I tried to make these examples as such that they are not dependent on Composer or any such 'mechanism', hence why I wrote my own little autoloader.

I might rework this to only have one autoload.php, but at the time I was thinking that maybe people wanted to test one set of examples, that is why I used the require_once on each of the examples.